src/tokens/transformation/web/color.js

```diff
@@ -0,0 +1,20 @@
+const TinyColor = require('@ctrl/tinycolor')
+
+module.exports = {
+  type: 'value',
+  matcher(token) {
+    return token.type === 'color'
+  },
+  transformer({ value, referencedVariable }) {
+    if (referencedVariable) {
+      const name = referencedVariable.substring(1).replaceAll('.', '-')
```

Comment:
Small point: This is the same as what we do with skia for referenced variables too, right (ignore the first char, and replace `.` with `-`? Should the knowledge of how the references variables translate to names be shared?